airdrop/getter.gno

```diff
@@ -0,0 +1,125 @@
+package airdrop
+
+import (
+	"std"
+	"strconv"
+	"strings"
+
+	"gno.land/p/demo/json"
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/demo/uint256"
+)
+
+// GetAirdropConfig returns the current configuration of the airdrop as a JSON string.
+//
+// Example:
+//
+//	{
+//	  "refundable_timestamp": "1234567890",
+//	  "refund_to": "g1234567890abcdefghijklmnop"
+//	}
+func (a *Airdrop) GetAirdropConfig() string {
```

Comment:
make it primitive

Reply:
https://github.com/gnoswap-labs/governance/pull/19/commits/a24003903330ffb7ba3e1f7b0180a6e6e8da951e

Reply:
All fixed to receive a JSON string as a parameter.

https://github.com/gnoswap-labs/governance/pull/19/commits/b1335b03623b74940851e9db25146e9218be526d